modules/monitoring/manifests/checks.pp

```diff
@@ -152,93 +152,95 @@
 
-  # Drop all but the last minute of data (assuming 5 second intervals)
-  $drop_first = '-12'
+    # Drop all but the last minute of data (assuming 5 second intervals)
+    $drop_first = '-12'
 
-  # Smooth out the data by using keepLastValue, with a limit of 132
-  # intervals, which is 10 minutes, plus a minute of padding to avoid
-  # any spurious alerts from late arriving metrics. It's important to
-  # set a limit here so that if the metrics are missing, then the
-  # alerts will fire.
-  $keep_last_value_limit = '132'
+    # Smooth out the data by using keepLastValue, with a limit of 132
+    # intervals, which is 10 minutes, plus a minute of padding to avoid
+    # any spurious alerts from late arriving metrics. It's important to
+    # set a limit here so that if the metrics are missing, then the
+    # alerts will fire.
+    $keep_last_value_limit = '132'
 
-  icinga::check::graphite { 'check_search_api_govuk_index_size_changed':
-    target              => "absolute(diffSeries(keepLastValue(stats.gauges.govuk.app.search-api.govuk_index.docs.count,${keep_last_value_limit}), timeShift(keepLastValue(stats.gauges.govuk.app.search-api.govuk_index.docs.count,${keep_last_value_limit}), \"7d\")))",
-    warning             => 3000,
-    critical            => 10000,
-    desc                => 'search-api govuk index size has significantly increased/decreased over the last 7 days',
-    host_name           => $::fqdn,
-    notification_period => 'inoffice',
-    action_url          => "https://grafana.${app_domain}/dashboard/file/search_api_index_size.json",
-    notes_url           => monitoring_docs_url(search-api-index-size-change),
-    from                => '25minutes',
-    args                => "--dropfirst ${drop_first}",
-  }
-  icinga::check::graphite { 'check_rummager_govuk_index_size_changed':
-    ensure    => 'absent',
-    target    => undef,
-    desc      => undef,
-    warning   => undef,
-    critical  => undef,
-    host_name => undef,
-  }
+    icinga::check::graphite { 'check_search_api_govuk_index_size_changed':
+      target              => "absolute(diffSeries(keepLastValue(stats.gauges.govuk.app.search-api.govuk_index.docs.count,${keep_last_value_limit}), timeShift(keepLastValue(stats.gauges.govuk.app.search-api.govuk_index.docs.count,${keep_last_value_limit}), \"7d\")))",
+      warning             => 3000,
+      critical            => 10000,
+      desc                => 'search-api govuk index size has significantly increased/decreased over the last 7 days',
+      host_name           => $::fqdn,
+      notification_period => 'inoffice',
+      action_url          => "https://grafana.${app_domain}/dashboard/file/search_api_index_size.json",
+      notes_url           => monitoring_docs_url(search-api-index-size-change),
+      from                => '25minutes',
+      args                => "--dropfirst ${drop_first}",
+    }
+    icinga::check::graphite { 'check_rummager_govuk_index_size_changed':
```

Comment:
These absent ones can be removed now, as puppet has run everywhere.

Reply:
will do, thanks.